coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Products/ProductResponse.cs

```diff
@@ -16,7 +17,11 @@ namespace CoffeeCard.Models.DataTransferObjects.v2.Products
     ///     "description": "Coffee clip card of 10 clips",
     ///     "isPerk": true,
     ///     "visible": true,
-    ///     "AllowedUserGroups": ["Manager", "Board"]
+    ///     "AllowedUserGroups": ["Manager", "Board"],
```

Comment:
Should be camelCase :)

Reply:
Addressed in fe39ffed2d7d416aee95f353586cf80e70afeb6f